MAUI/TreeMap/getting-started.md

```diff
@@ -0,0 +1,403 @@
+---
+layout: post
+title: Getting Started with .NET MAUI TreeMap | Syncfusion
+description: Learn about getting started with Syncfusion .NET MAUI TreeMap (SfTreeMap) in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Getting Started with .NET MAUI TreeMap (SfTreeMap)
+
+This section provides a quick overview of how to get started with the `.NET MAUI TreeMap (SfTreeMap)` for .NET MAUI and a walk-through to configure the .NET MAUI TreeMap in a real-time scenario.
+
+## Creating an Application using the .NET MAUI TreeMap
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add the SfTreeMap to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.TreeMap`, then install it.
+
+3. To initialize the control, import the control namespace `Syncfusion.Maui.TreeMap` in XAML or C# code.
+
+4. Initialize `SfTreeMap`.
+
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+        xmlns:treemap="clr-namespace:Syncfusion.Maui.TreeMap;assembly=Syncfusion.Maui.TreeMap">
+        <treemap:SfTreeMap />
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.TreeMap;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfTreeMap treeMap = new SfTreeMap();
+        this.Content = treeMap;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The [Syncfusion.Maui.Core](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.html) NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 8" %}
+
+    
+    using Syncfusion.Maui.Core.Hosting;
+    public static class MauiProgram
+    {
+	    public static MauiApp CreateMauiApp()
+	    {
+	        var builder = MauiApp.CreateBuilder();
+		    builder
+			    .ConfigureSyncfusionCore()
+			    .UseMauiApp<App>()
+			    .ConfigureFonts(fonts =>
+			    {
+				    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+				    fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
+			    });
+
+		    return builder.Build();
+	    }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Populating treemap items
+
+You can use `DataSource` property of SfTreeMap to populate the treemap items.
```

Comment:
While explaining DataSource,  need to explain about `PrimaryValuePath` property for calculating the TreeMap item size.

Reply:
Content changed